project/TastyMiMaFilters.scala

```diff
@@ -63,22 +63,28 @@ object TastyMiMaFilters {
     ProblemMatcher.make(ProblemKind.MissingTermMember, "scala.util.hashing.Hashing.fromFunction"),
     ProblemMatcher.make(ProblemKind.IncompatibleTypeChange, "scala.collection.IterableOnceOps.aggregate"),
     ProblemMatcher.make(ProblemKind.IncompatibleTypeChange, "scala.sys.package.addShutdownHook"),
+    ProblemMatcher.make(ProblemKind.IncompatibleTypeChange, "scala.collection.mutable.HashTable.init"),
+
+    // Probably OK: default parameter
+    ProblemMatcher.make(ProblemKind.MissingTermMember, "scala.*$default$*"),
 
-    // Problems with class constructors
+    // Problem: secondary constructors?
     ProblemMatcher.make(ProblemKind.MissingTermMember, "scala.*.<init>"),
-    ProblemMatcher.make(ProblemKind.IncompatibleTypeChange, "scala.*.<init>"),
+
+    // Problem The symbol scala.*.<init> has a more restrictive visibility qualifier in current version
+    // Only on primitive type final abstract classes
```

Comment:
If it's only on the primitive types, that's at most 9 things. Consider spelling them out explicitly so that you don't catch other constructors by mistake.

Reply:
Expanded